src/Terminus.php

```diff
@@ -137,11 +141,43 @@ public function run(InputInterface $input, OutputInterface $output)
     private function getCommands(array $options = ['path' => null, 'namespace' => null,])
     {
         $discovery = new CommandFileDiscovery();
-        $discovery->setSearchPattern('*Command.php')->setSearchLocations([]);
+        $discovery->setSearchPattern('*Command.php')->setSearchLocations([])->setSearchDepth(4);
```

Comment:
Actually this isn't needed anymore. The plugin discoverer uses a separate instance of CommandFileDiscovery so the core command searching can be left at the default of 2